README.md

```diff
@@ -30,17 +30,20 @@ Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pul
 
 To build and run the extension locally:
 
-1. `git clone git@github.com:MicroShed/mp-rest-client-generator-vscode-ext.git`
-2. `cd mp-rest-client-generator-vscode-ext`
-3. `npm install`
-4. Run the extension in VS Code by selecting `Run Extension` from the debug panel or by pressing `F5`
+1. [Fork](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) this repository
+2. Clone your fork: `git clone git@github.com:{your-username}/mp-rest-client-generator-vscode-ext.git`
+3. `cd mp-rest-client-generator-vscode-ext`
+4. Install dependencies: `npm install`
+5. Run the extension in VS Code by selecting `Run Extension` from the debug panel or by pressing `F5`
 
    Alternatively, build a `.vsix` file:
 
    - Run `vsce package` to generate the `mp-rest-client-generator-vscode-ext-xxx.vsix` file
    - Install the extension to VS Code by `View/Command Palette`
    - Select `Extensions: Install from VSIX...` and choose the generated `mp-rest-client-generator-vscode-ext-xxx.vsix` file
 
+6. Run the generate rest client command by right clicking on a folder and pressing "Generate a MicroProfile REST Client"
```

Comment:
Should be:

Run the extension by right-clicking on a folder and selecting "Generate a MicroProfile REST Client", or via the command palette by selecting "MicroProifle: Generate a MicroProfile REST Client"